metrics/meter.go

```diff
@@ -5,7 +5,7 @@ import (
 	"time"
 )
 
-// Meters count events to produce exponentially-weighted moving average rates
+// Meter methods count events to produce exponentially-weighted moving average rates
```

Comment:
Please don't add `methods` all over, it's a simple interface.

Reply:
Thank you - updated.